aws_lambda_powertools/logging/logger.py

```diff
@@ -292,6 +273,29 @@ def structure_logs(self, append: bool = False, **kwargs):
                 handler.setFormatter(JsonFormatter(**self._default_log_keys, **kwargs))
 
 
+def _get_log_level(level: Union[str, int]) -> Union[str, int]:
```

Comment:
make this a classmethod, not a separate function - it's still part of the encapsulated logger class logic